README.md

```diff
@@ -1,47 +1,66 @@
-# Tolgee browser plugin
+<h1 align="center" style="border-bottom: none">
+    <b>
+        <a href="https://chromewebstore.google.com/detail/tolgee-tools/hacnbapajkkfohnonhbmegojnddagfnj?hl=en">Tolgee Chrome Browser Plugin</a><br>
+    </b>
+ Modify your translations live and </br> take screenshots automatically
+</h1>
 
-[![twitter](https://img.shields.io/twitter/follow/Tolgee_i18n?style=social)](https://twitter.com/Tolgee_i18n)
-[![github stars](https://img.shields.io/github/stars/tolgee/chrome-plugin?style=social)](https://github.com/tolgee/chrome-plugin)
-[![slack](https://img.shields.io/badge/slack-Tolgee%20community-blue)](https://tolg.ee/slack)
+<div align="center"> 
 
 [<img src="https://raw.githubusercontent.com/tolgee/documentation/main/tolgee_logo_text.svg" alt="Tolgee" width="200" />](https://tolgee.io)
 
-Translate your web application more effectively with Tolgee plugin!
+Translate your web application more effectively with the Tolgee plugin!
+
+![licence](https://img.shields.io/badge/license-MIT-blue)
+[![github stars](https://img.shields.io/github/stars/tolgee/chrome-plugin?style=social)](https://github.com/tolgee/chrome-plugin)
+[![github stars](https://img.shields.io/github/stars/tolgee/tolgee-platform?style=social&label=Tolgee%20Platform)](https://github.com/tolgee/tolgee-platform)
+[![Github discussions](https://img.shields.io/github/discussions/tolgee/tolgee-platform)](https://github.com/tolgee/tolgee-platform/discussions)
+[![Read the Docs](https://img.shields.io/badge/Read%20the%20Docs-8CA1AF?logo=readthedocs&logoColor=fff)](https://docs.tolgee.io/)
+[![Slack](https://img.shields.io/badge/Slack-4A154B?logo=slack&logoColor=fff)](https://join.slack.com/t/tolgeecommunity/shared_invite/zt-2zp55d175-_agXTfKKVbf1BYXlKlmwbA)
+[![YouTube](https://img.shields.io/badge/YouTube-%23FF0000.svg?logo=YouTube&logoColor=white)](https://www.youtube.com/@tolgee)
+[![LinkedIn](https://custom-icon-badges.demolab.com/badge/LinkedIn-0A66C2?logo=linkedin-white&logoColor=fff)](https://www.linkedin.com/company/tolgee/)
+[![X](https://img.shields.io/badge/X-%23000000.svg?logo=X&logoColor=white)](https://x.com/Tolgee_i18n)
+
+</div>
+
+## Tolgee Chrome plugin is an integral part of Tolgee ecosystem
 
 <img src="images/tolgee-chrome-translate-screenshot.png" alt="Tolgee Translate Content Instantly" width="500">
 
-Tolgee Chrome plugin is an integral part of Tolgee ecosystem. It enables tolgee-js to take screenshots automatically.
+It enables tolgee-js to take screenshots automatically. With the Tolgee plugin, you can modify translations live on your page so you can see the location of all translations. Find out how exactly the page will look with the newly added translation. 
```

Comment:
I would remove this part. It's not precise and it's actually described in features section.